.travis.yml

```diff
@@ -6,10 +6,11 @@ python:
   - "3.6"
 
 install:
-  - pip install -r tests/requirements.txt
-
-run_app:
-  - python web/app.py
+  - pip install -r requirements.txt
+  - pip install -r requirements-integration.txt
+  - pip install -r requirements-unit.txt
 
 script:
-  - python integration-test/test.py
\ No newline at end of file
+  - export SQLALCHEMY_DATABASE_URI='sqlite:///web.db'
```

Comment:
Exporting the environment variable seems to be necessary in the test

Otherwise, it will trigger the 
```
AttributeError: 'NoneType' object has no attribute 'drivername'
```
Error

Reply:
I suspect that this might need to explicitly import dotenv, and load all the environment variables.  (I would prefer this solution rather than manually exporting all the necessary environment variables.) It could turn out that the testing environment variables are significantly different from the app environment variables, in which case I would prefer using a `.env_test` file or similar.